keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1188,239 +937,309 @@ drivers are expected to be written for Kubernetes.
 
 ##### ResourceSlice
 
-For each node, one or more ResourceSlice objects get created. The kubelet
-publishes them with the node as the owner, so they get deleted when a node goes
+For each node, one or more ResourceSlice objects get created. The drivers
+on a node publish them with the node as the owner, so they get deleted when a node goes
 down and then gets removed.
 
 All list types are atomic because that makes tracking the owner for
 server-side-apply (SSA) simpler. Patching individual list elements is not
-needed and there is a single owner (kubelet).
+needed and there is a single owner.
 
 ```go
-// ResourceSlice provides information about available
-// resources on individual nodes.
+// One of more slices represent a pool of devices managed by a given driver. For
+// node-local devices, each node is a pool. How many slices the driver uses
+// to publish that pool is driver-specific. Within each pool the device name must
+// be unique.
```

Comment:
> Within each pool the device name must be unique.

This is confusing to me -- why would the unique name not be called "pool name" in this case?

Reply:
Or are you trying to say: "Each device in a given pool must have a unique name"?

Reply:
It's "each device in a given pool must have a unique name".